player/command.c

```diff
@@ -2856,6 +2857,18 @@ static int mp_property_sub_text(void *ctx, struct m_property *prop,
     return M_PROPERTY_NOT_IMPLEMENTED;
 }
 
+static int mp_property_sub_text(void *ctx, struct m_property *prop,
+                                int action, void *arg)
+{
+    return get_sub_text(ctx, prop, action, arg, 0);
+}
+
+static int mp_property_secondary_sub_text(void *ctx, struct m_property *prop,
+                                int action, void *arg)
```

Comment:
Spacing here too.

Reply:
By spacing, did you mean making the `mp_property_secondary_sub_text` and `mp_property_sub_text` functions into one-liners?